plugins/catalog-react/src/components/EntityAutocompletePicker/EntityAutocompletePicker.tsx

```diff
@@ -74,17 +76,26 @@ export function EntityAutocompletePicker<
   } = useEntityList<T>();
 
   const catalogApi = useApi(catalogApiRef);
+  const availableValuesFilters = filtersForAvailableValues.map(
+    f => filters[f] as EntityFilter | undefined,
+  );
   const { value: availableValues } = useAsync(async () => {
     const facet = path;
     const { facets } = await catalogApi.getEntityFacets({
       facets: [facet],
-      filter: filters.kind?.getCatalogFilters(),
+      filter: availableValuesFilters
```

Comment:
Related from the original PR, https://github.com/backstage/backstage/pull/15112/files#r1050126893. 

We can also update this block to [`reduceCatalogBackendFilters`](https://github.com/backstage/backstage/blob/master/plugins/catalog-react/src/utils/filters.ts#L48-L71).

Reply:
Ah, didn't see that util, I updated the PR to use the available function. Thanks!